cloudant-sync-datastore-core/src/main/java/com/cloudant/sync/replication/GetRevisionTaskThreaded.java

```diff
@@ -121,6 +122,8 @@ public GetRevisionTaskThreaded(CouchDB sourceDb,
                 DocumentRevsList>(executorService, this.requests, threads + 1) {
             @Override
             public DocumentRevsList executeRequest(BulkGetRequest request) {
+                // since this is part of a thread pool, we'll rename each thread as it takes a task.
+                Thread.currentThread().setName("GetRevisionThread: "+GetRevisionTaskThreaded.this.sourceDb.getIdentifier());
```

Comment:
Formatter is not automatic by default, you can set it to run on all lines or modified lines.